api/envoy/extensions/filters/network/mysql_proxy/v3/mysql_proxy.proto

```diff
@@ -15,15 +18,68 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // MySQL Proxy :ref:`configuration overview <config_network_filters_mysql_proxy>`.
 // [#extension: envoy.filters.network.mysql_proxy]
 
+// Auth information of MySQL.
+message AuthInfo {
+  config.core.v3.DataSource auth_username = 1 [(udpa.annotations.sensitive) = true];
+
+  config.core.v3.DataSource auth_password = 2 [(udpa.annotations.sensitive) = true];
```

Comment:
can be just username/password (without `auth_` prefix).

Reply:
fix.